reana_db/models.py

```diff
@@ -178,6 +178,7 @@ class User(Base, Timestamp, QuotaBase):
         sync_backref=False,
     )
     audit_logs = relationship("AuditLog", backref="user_")
+    idp_id = Column(String(length=255))
```

Comment:
Please note that changing the data model would require us to add DB schema migration scripts. We use Alembic for this. Here is the recipe how to generate one: https://github.com/reanahub/reana/wiki/Tips-for-SQLAlchemy#autogenerate-an-alembic-migration